### ISSUE TITLE
Cypress check_links.js times out for Apple Store and Instagram

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#2969]
## Where to find the issue

[cypress/integration/hybrid/check_links.js](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/integration/hybrid/check_links.js)

## Describe the issue

When running the Cypress test [cypress/integration/hybrid/check_links.js](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/integration/hybrid/check_links.js) to check for correct working of hyperlinks, the test sometimes fails accessing some external links.

The timeout do not always occur. The following links are affected

- https://apps.apple.com/de/app/116117-app/id1465237675
- https://apps.apple.com/de/app/corona-warn-app/id1512595757
- https://www.instagram.com/rki_fuer_euch/

## Steps to reproduce

Execute
`npx cypress run -s cypress/integration/hybrid/check_links.js -c baseUrl=https://coronawarn.app --headed --browser chrome`
in root directory of `cwa-website` clone.

Example error:

```
Method: GET
URL: https://apps.apple.com/de/app/corona-warn-app/id1512595757

No response was received within the timeout.
```

## Suggested change

Add the following links to the list on
https://github.com/corona-warn-app/cwa-website/blob/3c204036c9368378daca86478cb283b5b29cd828/cypress/integration/hybrid/check_links.js#L37

- https://apps.apple.com/de/app/116117-app/id1465237675
- https://apps.apple.com/de/app/corona-warn-app/id1512595757
- https://www.instagram.com/rki_fuer_euch/

which avoids the listed links being tested.

---
Internal Tracking ID: [EXPOSUREAPP-13420](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13420)

**Comment by MikeMcC399** [comment]
Other links to apple, such as https://support.apple.com/en-gb/HT201252 also seem to be affected by occasional time-out issues.

**Comment by brifemu** [comment]
Hi @MikeMcC399, I'm working on this now. I was thinking that maybe is better to avoid all links that include 'apple.com' in the URL and also social media URLs (Twitter and Instagram profiles). What do you think about it?

The reason to avoid social media URLs is that they exist in almost all pages and is easier to get blocked cause of the flood.

**Comment by MikeMcC399** [comment]
Hi @brifemu 

I have only seen problems with apple and instagram so I think it is sufficient just to exclude the links which showed errors in the past. I didn't see any problem with Twitter so far.

It is only possible to build up experience over time about which links cause a problem in this area because they only fail sometimes, not always. Also, only some of the apple links fail, so that could depend on the timing of each request. This is not really an exact science, more of a trial-and-error process!

**Comment by brifemu** [comment]
@MikeMcC399 Perfect! I'm going to create the PR then.

**Comment by MikeMcC399** [comment]
- This issue is resolved by PR #2973.
